INSTALL.md

```diff
@@ -15,7 +15,7 @@
 After checking out the source from github with git submodules is is
 possibleto install the build tools with GNU Guix
 
-    guix package -i gcc-toolchain gdb bash ld-wrapper ldc which python2 git
+    guix package -i gcc-toolchain gdb bash ld-wrapper ldc which python3 git
```

Comment:
```suggestion
    guix package -i gcc-toolchain gdb bash ld-wrapper ldc which python git
```

In GNU Guix, `python` is Python 3, Python 2 requires an explicit `python2`:

```sh
guix package -A python | grep -v '-'
```

```
micropython     1.11    out     gnu/packages/python.scm:536:2
python  3.7.4   out,tk  gnu/packages/python.scm:327:2
python2 2.7.16  out,tk  gnu/packages/python.scm:98:2
```

Reply:
> In GNU Guix, python is Python 3, Python 2 requires an explicit python2:

Neat! Very soon it looks like this will be the case everywhere.